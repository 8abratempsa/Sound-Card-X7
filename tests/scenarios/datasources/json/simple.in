octosql "SELECT COUNT(*), SUM(field2), SUM(len(field3)) FROM fixtures/objects.json"
