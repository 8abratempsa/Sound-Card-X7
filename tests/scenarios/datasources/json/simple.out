+-------+------------+-------+
| count | sum_field2 |  sum  |
+-------+------------+-------+
| 20500 |     763625 | 92250 |
+-------+------------+-------+
